GUIDELINES.adoc

```diff
@@ -33,7 +33,7 @@ Use the following template for glossary entries:
 * `\image:images/caution.png[with caution]`
 |`<term>`               |The term to be described.          |
 |`<class>`              |The part of speech of the term.|Possible values: noun, verb, adjective, adverb, pronoun, conjunction, preposition. Use noun for acronyms or abbreviations. Leave the field empty for symbols.
-|`<description>`        |The description of the term.|Use full sentences with periods. If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
+|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should use the term in this form. If we should not use the term, explain why. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

Comment:
```suggestion
|`<description>`        |The description of the term.|Use full sentences with periods. If a term is specific to a particular product, start the description with "In <product_name>,". If a term has more than one meaning, use numbers to separate them. For example: (1) <first-meaning>. (2) <second-meaning>. Add information about why we should or shouldn't use the term in this form. Alternatively, explain why the terms listed in the *Incorrect forms* field are incorrect.
```

Reply:
Thanks @mportman12 that works, I'll make that change